mm2src/coins/utxo/rpc_clients.rs

```diff
@@ -2010,9 +2011,12 @@ impl ElectrumClient {
     // get_tx_height_from_storage is always preferred to be used instead of this, but if there is no headers in storage (storing headers is not enabled)
     // this function can be used instead
     async fn get_tx_height_from_rpc(&self, tx: &UtxoTx) -> Result<u64, GetTxHeightError> {
+        // Refactor scripthash_get_history to avoid `capturing dynamic environment in a fn item" when using mockable
+        // attribute
+        let scripthash_get_history = |s: String| async move { self.scripthash_get_history(s.as_str()).compat().await };
```

Comment:
Taking the comment above into account, this is not needed anymore. Also this problem happens due to this async function is declared as mockable, but it shouldn't be (except `async_trait` only).

Reply:
got that. 🙏🏼